app/entity-repositories/pasteRepository.js

```diff
@@ -0,0 +1,92 @@
+class PasteReository {
+    constructor ({pool}) {
+        this.pool = pool;
```

Comment:
We need to hide this behind some kind of `DatabaseProvider` with `execute` method. Pool-related operations is not a part of Repo